### DISCUSSION TITLE
Secret algorithms and secret intentions (thinking cap)

### DISCUSSION THREAD

**Discussion by rh5140** [albertkun/21S-ASIAAM-191A#97] [Past Thinking Caps]
Closed source tools are harmful because users won’t be able to fully understand how they work and what biases may be embedded in them. In particular, I’m thinking of the algorithms that control what we see online.

I was going to say that users don’t consent to their data being used in this way by corporations, but I realized that I have never closely read any of the EULAs I agreed to… Even if users do technically sign away the right to their data and give corporations the right to do whatever with it, legal doesn’t equal ethical, and I think users should at the very least be allowed to understand how their data is used.

This [article](https://www.wired.com/story/one-mans-obsessive-fight-to-reclaim-his-cambridge-analytica-data/) covers how David Carroll took Cambridge Analytica’s parent company to court to reclaim his data, and they plead guilty because of UK data protection laws even though Carroll is an American citizen. Carroll wanted to show the world how little control Americans have over their own data and how it’s possible for things to be different, but it must have also put those who exploit user data on high alert instead.
> “No campaign is making the mistake of exporting data to the U.K. again, so in 2020 we will have no rights,” [Carroll] said. One exception: California residents who can avail themselves of disclosure and deletion rights in the California Consumer Privacy Act.

(from [The Real Problem Wasn’t Cambridge Analytica, But The Data Brokers That Outlived It (Forbes)](https://www.forbes.com/sites/robpegoraro/2020/10/08/the-real-problem-wasnt-cambridge-analytica-but-the-data-brokers-that-outlived-it))

This [paper](https://hdsr.mitpress.mit.edu/pub/7z10o269/release/4) focuses on the lack of transparency in the algorithms used for predictive policing. It suggests that if users knew what was going into algorithms, they may find that those algorithms are useless.
> One such secret algorithm—called COMPAS—is widely-used across the justice system, and has been the subject of high-profile lawsuits. But what do secret algorithms like COMPAS actually compute? If we could peek inside, perhaps we could determine what makes these algorithms tick—and whether we need them at all.

Something analogous to issues with closed source is [Apple’s (and Asus’s and other companies’) monopoly on even repairs for their products](https://www.vox.com/the-goods/2019/7/3/18761691/right-to-repair-computers-phones-car-mechanics-apple). By making the only legal means of repair through Apple, the company can hike up the price for repairs and incentivize consumers to continue to purchase new products. I think closed source can also result in monopolizing behavior like that, but I’m not certain that it would be a major factor.

I don’t have as much to say on flaws of open source other than that there’s no way to make sure the intentions of future users aren’t harmful.

**Comment by skarakas1** [comment]
Great comment! I think two of the defining political concepts of our time will be the balance between transparency (in this case, with algorithms) and privacy. I think that a big part of the problem here is conceptualizations of corporate personhood (Citizens United being one example). I believe that corporations should have to be transparent, while user data privacy should have more legal protection.
I think that predictive policing is a great example of why transparency is necessary. It is one of the most dangerous issues on the horizon. At the surface it sounds like a computer-assisted version of profiling, whether that profiling is racial or perhaps geographical or class-based. Recently I have seen a few news articles and opinion pieces extolling the virtues of digital law enforcement (usually without using the words 'predictive policing') as a potential method for de-emphasizing the role of police officers.
While I am completely in support of rethinking the police, significantly reducing their budgets, or even completely replacing them with other relevant community resources, the idea of replacing the police with CCTV, predictive algorithms, and facial recognition seems like it would almost certainly have catastrophic unintended consequences. Combined with the privatization of most aspects of the digital world and neoliberal government tendencies to subcontract (for instance private prisons) it seems like we could easily find ourselves in a crisis of corporate governmentality and panopticism. My understanding is that something like this is already happening in China with facial recognition and social credit scores, although I don't know this from firsthand experience.
